### PR TITLE
Update accounts-2018.html

### PR DESCRIPTION
**Pull request by vchernin**
## Description
Currently this page has adjust.com links in it. adjust is a self-proclaimed "mobile measurement company", which is English for just another company trying to monetize user data. Mozilla is better than this. Please give the direct download links to apps instead of routing them through services such as adjust.

### CODE REVIEW

**Comment by vchernin** [comment]
@

**Comment by alexgibson** [comment]
Thank you for opening this issue and bringing it to our attention. I reached out to our marketing contact on the use of these links and they provided some information that hopefully helps to clarify Mozilla's use of Adjust:

> Adjust links are used for attribution of mobile installs, or in simpler terms, the identification of sources for installs of mobile applications. Adjust associates mobile installs with specific sources, similar to UTM parameters which are commonly used to associate desktop installs with specific sources.
>
> For a little more context on how Adjust data is used by the Marketing and Product organizations:
> 1. If installs attributed to an article talking about Firefox Lite result in lower retention of users who install Lite after reading the article, we would look to improve the article.
> 2. Adjust links are used for paid media, which allows for the evaluation of the success of these paid media campaign, and distinguishes paid media installs from organic or Product Marketing installs (such as those from Moz.org).
>
> I deeply appreciate your concerns about data collection, especially that facilitated by another organization. Monetization of user data is antithetical to our mission, and expressly outside the scope of any current or planned utilization of Adjust."

I also found some further information on the use of Adjust in our [privacy policy](https://www.mozilla.org/en-US/privacy/firefox/). You can find the relevant section under expandable "Measure and support our marketing" heading. I copied it for reference. You can find out more info by visiting the privacy policy and following the related links.

> On iOS and Android: Firefox by default sends mobile campaign data to Adjust, our analytics vendor, which has its own privacy policy. Mobile campaign data includes a Google advertising ID, IP address, timestamp, country, language/locale, operating system, and app version. Read the documentation.

Monetizing user data is never something Mozilla would intend to do with a service such as Adjust, and if it we're happening contrary to our knowledge then we could of course reevaluate our current usage policy. Mozilla does not take use of third party SDK's lightly, and whichever services we do use have undergone a large amount of scruitiny by both our privacy and legal teams prior to being used.

I'm going to close this issue, but thank you again for raising it.
